.github/workflows/wikiman.validate.yml

```diff
@@ -0,0 +1,17 @@
+name: wikiman.yaml validation
+
+on: push
```

Comment:
Why only on push?

Reply:
This means any push to any branch, which, as far as I can tell, is more often than some of the other actions, which only run on the main branch and on PRs. The only difference (I think?) is that this also makes it run on branches before you open a PR. FWIW https://github.com/wbstack/mediawiki/blob/main/.github/workflows/wbstack.yaml.lint.yml#L3 (which I also added :P) does  the same.

Reply:
Changed it!